front-end/src/components/newsfeed-view/news-list/news-card/TwitterNewsCard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Timestamp from './elements/Timestamp';
+import './newscard.css';
+
+const TwitterNewsCard = ({data}) => (
+    <div className="news_card twitter">
+    <div className = "content">
+        <div className="source">
+            <div className="author">
+                <img src = "https://pbs.twimg.com/profile_images/559709902990348288/NwYEyKfg_400x400.jpeg" />
```

Comment:
hard coded value. Mostlikely somebody forgot this

Reply:
It didn't make sense to use any variables cause I didn't have any feed structure from backend. But ofc it will be fixed

Reply:
you can have code that transform data later (or have to anyway). Wouldnt be a problem

producer (api returns) -> transformer (our code) -> consumer (react component, arbitrary choice up to you).
So api struc changes, you only need to change the transformer code, which is rather easy to do